src/timecode/Timecode.cs

```diff
@@ -490,6 +491,73 @@ private void SetTotalFramesUsingDropFrames(int hours, int minutes, int seconds,
       int totalMinutes = (60 * hours) + minutes;
       int totalFrames = ((hourFrames * hours) + (minuteFrames * minutes) + (timeBase * seconds) + frames) - (dropFrames * (totalMinutes - (totalMinutes / 10)));
       TotalFrames = totalFrames;
+    }
+
+    #region staticMethods
+

```

Comment:
These additions are great. We do need to implement a static call for the method ConvertFramerate() (such as the provided initial example in issue #20). I will assist you by writing a suggestion which you can add directly using your Github, or by updating your code and push a new commit. See below for the suggestion:

```suggestion
/// <summary>
    /// Converts a timecode string to a timecode string of a different framerate.
    /// </summary>
    /// <param name="originalTimecode">The original timecode, formatted "HH:MM:SS:FF" or "-HH:MM:SS:FF".</param>
    /// <param name="originalFramerate">The original timecode framerate.</param>
    /// <param name="destinationFramerate">The target framerate to convert to.</param>
    /// <returns></returns>
    /// <exception cref="ArgumentException"></exception>
    public static string ConvertFramerate(string originalTimecode, Framerate originalFramerate, Framerate destinationFramerate)
    {
      // Validate the original timecode format
      Regex tcRegex = new Regex(RegexPattern);
      if (!tcRegex.IsMatch(originalTimecode))
        throw new ArgumentException("Invalid timecode format.", nameof(originalTimecode));

      Timecode timecode = new Timecode(originalTimecode, originalFramerate);
      timecode.ConvertFramerate(destinationFramerate);
      return timecode.ToString();
    }
```

Reply:
@RasmusBroborg Above review comments applied and committed the changes